src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/UIHintAttribute.cs

```diff
@@ -92,12 +92,12 @@ public UIHintImplementation(string uiHint, string presentationLayer, params obje
             /// <summary>
             ///     Gets the name of the presentation layer that supports the control type in <see cref="UIHint" />
             /// </summary>
-            public string PresentationLayer { get; }
+            public string? PresentationLayer { get; }
 
             // Lazy load the dictionary. It's fine if this method executes multiple times in stress scenarios.
             // If the method throws (indicating that the input params are invalid) this property will throw
             // every time it's accessed.
-            public IDictionary<string, object> ControlParameters =>
```

Comment:
cc: @GrabYourPitchforks for GetHashCode below

Reply:
Anything in particular we need to do here?